src/Lean/Elab/Tactic/BuiltinTactic.lean

```diff
@@ -120,7 +123,7 @@ where
     Term.withNarrowedTacticReuse (fun stx => (stx[0], mkNullNode stx.getArgs[1:])) goEven stx
 
 @[builtin_tactic seq1] def evalSeq1 : Tactic := fun stx =>
-  evalSepTactics stx[0]
+  (evalSepTactics) stx[0]
```

Comment:
Please just pass the argument explicitly, in which case we may as well remove the default value